README.md

```diff
@@ -2,31 +2,111 @@
+
+4. A prompt to import all of the files of the OneSignal Unity SDK will appear. Click **Import** to continue and compile the scripts into your project.
+5. Navigate to **Window > OneSignal** (or follow the popup if upgrading) in the Unity Editor which will bring up a window with some final steps which need 
+   to be completed in order to finalize the installation. The most important of these steps is **Import OneSignal packages**.
+   
+    > *Depending on your project configuration and if you are upgrading from a previous version, some of these steps may already be marked as "completed"*
+   
+    ![sdk setup steps window](Documentation/setup_window.png)
+
+6. After importing the packages Unity will notify you that a new registry has been added and the **OneSignal SDK Setup** window will have refreshed with a few additional 
+   steps. Following these will finalize your installation of the OneSignal Unity SDK.
+</details>
+
+<details>
+    <summary>Unity Package Manager</summary>
+1. From within the Unity Editor navigate to **Edit > Project Settings** and then to the **Package Manager** settings tab.
+   
+   ![unity registry manager](Documentation/package_manager_tab.png)
+
+2. Create a *New Scoped Registry* by entering 
+    ```
+    Name        npmjs
+    URL         https://registry.npmjs.org
+    Scope(s)    com.onesignal
+    ```
+   and click **Save**.
+3. Open the **Window > Package Manager** and switch to **My Registries** via the **Packages:** dropdown menu. You will see all of the OneSignal Unity SDK packages available
+   on which you can then click **Install** for the platforms you would like to include. Dependencies will be added automatically.
+4. Once the packages have finished importing you will find a new menu under **Window > OneSignal**. Open it and you will find some final steps which need to be completed
+   in order to finalize the installation.
+
+   > *Depending on your project configuration and if you are upgrading from a previous version, some of these steps may already be marked as "completed"*
 
-For account issues and support please contact OneSignal support from the [OneSignal.com](https://onesignal.com) dashboard.
+   ![my registries menu selection](Documentation/registry_menu.png)
+
+</details>
+
+## Platform Configuration
+### [iOS](https://documentation.onesignal.com/docs/unity-sdk-setup#step-5---ios-setup)
+### [Android](https://documentation.onesignal.com/docs/unity-sdk-setup#step-6---android-setup)
+
+## Usage
+You can find a complete implementation in our included [example MonoBehaviour](OneSignalExample/Assets/OneSignal/Example/Scripts/OneSignalExampleBehaviour.cs). Additionally we have included a
+[sample scene](OneSignalExample/Assets/OneSignal/Example/Scenes/OneSignalExampleScene.unity) which you can run to test out the SDK.
+
+### Initialization
+To get started add the following code in an appropriate place as your application is starting up.
```

Comment:
What's the appropriate place? Specifically, the file name and name of the method.

Reply:
There isn't a good answer to that. There is no "main" entry point in a Unity application. It's entirely up to the developer.

Reply:
Where would you recommend it be placed? I think an example would be helpful, unless the assumption that only people who already know this will use this package. I think it's safe to have an opinion here.

Reply:
I can provide a vague recommendation, ie:
> To get started add the following code in an appropriate place such as the `Start` method of a `MonoBehaviour` early in your application's lifecycle.

Reply:
Change in 4548489823f479b1e08266e3d51e9cea34e1ce9d